crates/oxc_linter/src/rules/jsx_a11y/prefer_tag_over_role.rs

```diff
@@ -0,0 +1,137 @@
+use std::collections::HashMap;
+
+use crate::{context::LintContext, rule::Rule, utils::has_jsx_prop_lowercase, AstNode};
+use oxc_ast::{
+    ast::{JSXAttributeItem, JSXAttributeValue, JSXElementName},
+    AstKind,
+};
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+
+#[derive(Debug, Error, Diagnostic)]
+#[error("eslint-plugin-jsx-a11y/prefer-tag-over-role: Prefer using semantic HTML tags over `role` attribute.")]
```

Comment:
```suggestion
#[error("eslint-plugin-jsx-a11y(prefer-tag-over-role): Prefer using semantic HTML tags over `role` attribute.")]

Reply:
Thanks!!
Fixed in [fix: text](https://github.com/oxc-project/oxc/pull/1831/commits/a44bf49732f8f80221695f9068d5e5f0e5e96caf) !!